common/src/main/java/com/wynntils/features/ItemGuessFeature.java

```diff
@@ -5,38 +5,19 @@
 package com.wynntils.features;
 
 import com.google.common.collect.ImmutableList;
-import com.wynntils.core.Reference;
-import com.wynntils.core.WynntilsMod;
 import com.wynntils.core.features.Feature;
 import com.wynntils.core.features.properties.FeatureInfo;
 import com.wynntils.core.features.properties.GameplayImpact;
 import com.wynntils.core.features.properties.PerformanceImpact;
 import com.wynntils.core.features.properties.Stability;
 import com.wynntils.core.webapi.WebManager;
-import com.wynntils.core.webapi.profiles.ItemGuessProfile;
-import com.wynntils.mc.event.ItemsReceivedEvent;
-import com.wynntils.mc.utils.ComponentUtils;
-import com.wynntils.mc.utils.ItemUtils;
-import com.wynntils.utils.reference.EmeraldSymbols;
-import com.wynntils.wc.objects.items.ItemProfile;
-import com.wynntils.wc.objects.items.ItemTier;
-import com.wynntils.wc.objects.items.ItemType;
-import com.wynntils.wc.utils.WynnItemMatchers;
-import com.wynntils.wc.utils.WynnUtils;
-import java.util.List;
-import java.util.Map;
-import net.minecraft.ChatFormatting;
-import net.minecraft.nbt.ListTag;
-import net.minecraft.nbt.Tag;
 import net.minecraft.network.chat.MutableComponent;
 import net.minecraft.network.chat.TranslatableComponent;
-import net.minecraft.world.item.ItemStack;
-import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 @FeatureInfo(performance = PerformanceImpact.SMALL, gameplay = GameplayImpact.LARGE, stability = Stability.STABLE)
 public class ItemGuessFeature extends Feature {
 
-    private static final boolean showGuessesPrice = true;
+    public static final boolean showGuessesPrice = true;
```

Comment:
Why is this public? Public fields are dubious in the first case, but even more so for Features. I see them as self-contained units, that no one should depend on.

Reply:
all the public fields are just placeholders for config values, however we end up implementing those

Reply:
Aha, I see. Maybe a better way to mark them would be to name them with a prefix, like `confShowGuessesPrice`. That would help with automating finding and fixing them up when we get a proper config framework. (And in the meantime it will make it a bit more clear, I think).

I'm not requiring that you do that in this PR (it's large enough as it is), but maybe back out the private -> public conversions, if you agree that a name prefix is a better idea.